app/src/main/kotlin/io/homeassistant/companion/android/launch/LaunchPresenterBase.kt

```diff
@@ -10,7 +11,8 @@ import kotlinx.coroutines.launch
 
 abstract class LaunchPresenterBase(
     private val view: LaunchView,
-    internal val serverManager: ServerManager
+    internal val serverManager: ServerManager,
+    internal val pushManager: PushManager
```

Comment:
I think the visibility of the attributes of this class needs to be lower to `protected` not `internal`.